### PR TITLE
Template option for network and storage mappings

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 2.5

[No Jira] Adds instructions on how to save network mappings and storage mappings as templates.

Previews:

1. https://file.emea.redhat.com/rhoch/template/html-single/#creating-network-mapping_mtv [Network mapping]
2. https://file.emea.redhat.com/rhoch/template/html-single/#creating-storage-mapping_mtv [Storage mapping]

### CODE REVIEW

**Comment by RichardHoch** [comment]
@ahadas anyone need to review this besides you, @yaacov, and @qiyuann ?

**Review by yaacov — APPROVED** [review]
lgtm

File: documentation/modules/creating-network-mapping.adoc [hunk 1]
```diff
@@ -26,6 +26,7 @@ You can create one or more network mappings by using the {ocp} web console to ma
 The *Source networks* and *Target namespaces/networks* text boxes become active.
 
 . Select a source network and a target namespace/network from the list.
+. Optional: To save your mapping as a template, select *Save current mapping as a template* and enter a name for your mapping.
```
Comment (qiyuann) [inline documentation/modules/creating-network-mapping.adoc:29]: Should this `Save current mapping as a template` option be added here? Is not on the `Create NetworkMap` window of `Migration -> NetworkMaps for virtualization`.

Comment (ahadas) [reply documentation/modules/creating-network-mapping.adoc:29]: yeah, I believe @qiyuann is right - every mapping that is created from the `NetworkMaps for virtualization` page produced a "mapping template". this option is supposed to appear only in the plan creation page/wizard

**Review by ahadas — COMMENTED** [review]
I see that the creation of the mapping template is [already mentioned in the migration plan page](https://github.com/kubev2v/forklift-documentation/blob/main/documentation/modules/creating-migration-plan.adoc?plain=1#L48-L58), so the changes in this PR are not needed

**Comment by RichardHoch** [comment]
This PR is not needed after all...

**Pull request closed without merging**